### ISSUE TITLE
Audit the table headers

### ISSUE THREAD

**Issue by hbillings** [18F/cms-hitech-apd#1469]
With all of the table rejiggering that's happened lately, we should do a full-page audit of the table headers to make sure they're coherent and consistent.

### This task is done when...
 - [ ] all of the tables have been audited and any new errors have been corrected
 - [ ] the header cells are associated with the correct data
 - [ ] screen readers read tables coherently
 - [ ] any content that is currently in what should be empty cells has been removed (see #1480 for examples)
 - [ ] Instead of `subtotal`, use `total` across the board for tables that are now broken apart by FFY

**Comment by hbillings** [comment]
This is mostly obsoleted by other tickets that have been opened.
